### ISSUE TITLE
blog post title link

### ISSUE THREAD

**Issue by SerGun78** [templaza/astroid-framework#48]
Good day! I can not properly configure the display of the title link of the materials in the blog, so that it is not displayed in the material itself.
It turns out only like this:
1.or the title of the material in the form of a link is both in the blog and already in the material
2. either there is no title in the blog in the form of a link, the transition to the material is only by the "more" button, there is no title in the material either.
This is for proper SEO.

**Comment by Joomlaplates** [comment]
Hello, 
you talk about the "article title" ?

**Comment by SerGun78** [comment]
Yes, sure

**Comment by Joomlaplates** [comment]
Same problem here?
https://github.com/templaza/astroid-framework/issues/23

**Comment by SerGun78** [comment]
yes, this problem. but it is only seen on astroid templates. Cassiopea, helix ultimate has no such problem.

**Comment by sonvnn** [comment]
Hi @Joomlaplates ,

I switched off title "Hide" in Article detail option and my article title is disappear. Please check my demo here http://joomla.templaza.com/astroid/joomla/index.php

It's the first article. You can check it. Others display normally. So this feature works well on my demo.

Hi @SerGun78 ,
I check title link in article detail. It's still work. No link in title of article detail http://joomla.templaza.com/astroid/joomla/index.php/8-business/33-ten-business-that-will-make-your-life-better

I am not sure about your issue. Please detail it.

Best Regards,
Sonny

**Comment by Joomlaplates** [comment]
**The problem is:**
if you hide the headline in article options for only one article, then the article headline will not shown in blog article view.
In blog view of articles, the headline of articles has to be shown, even if one headline in article details is set to "hide"

**Comment by SerGun78** [comment]
It should be like this:
1.blog
![изображение](https://user-images.githubusercontent.com/95705148/145705814-3ef83054-f252-42b8-b7b2-da8f90123aeb.png)
2. in every article
![изображение](https://user-images.githubusercontent.com/95705148/145705851-656c027a-7c8f-450f-9805-a5144a3f41d8.png)

**Comment by sonvnn** [comment]
Hi @Joomlaplates ,
Thank you I got it. I'm still researching this issue and haven't found the cause yet. I will update soon.

Hi @SerGun78 ,
That is Template area. You can custom and change it in your template. The framework is like an arsenal and the Template decides what it takes to fight.

Thanks & Best Regards,
Sonny

**Comment by SerGun78** [comment]
Sonny, show me how you can get it set up here: http://joomla.templaza.com/astroid/joomla/

I have already tried everything on the ONE and ZERO templates. Nothing succeeded.
But the standard Cassiopea template and Helix Ultimate have no such problem, everything is displayed correctly.
Those. or the template itself and the Astroid framework somehow override the output of this data to the frontend.

**Comment by sonvnn** [comment]
Hi @SerGun78 ,

Let's check "how to override the output from the Joomla! core" here https://docs.joomla.org/How_to_override_the_output_from_the_Joomla!_core

Thanks,

**Comment by SerGun78** [comment]
I don’t even want to understand this. As a user, I don't need this. Why change php (blog, article) so that it changes the core structure?

**Comment by sonvnn** [comment]
@SerGun78 I confirmed this issue from @Joomlaplates . I am investing about it and update in the next versions.

**Comment by Joomlaplates** [comment]
Hi Sonny, its a problem in "html" override from theme?
But I couldnt find the problem there.

**Comment by sonvnn** [comment]
Hi @Joomlaplates,
Yes, I am researching about this problem. But still don't know why. It's funny =))

**Comment by SerGun78** [comment]
the problem is in blog_item.php

**Comment by Joomlaplates** [comment]
> the problem is in blog_item.php

No, I dont think so.
The title is loading the org Joomla file, see pic.
![blog](https://user-images.githubusercontent.com/71080509/146538698-8b856336-1260-4290-aadd-0f77915c2d1b.jpg)

**Comment by SerGun78** [comment]
after replacing it with the original from the core, the title with the link appeared on the blog. Verified personally.

**Comment by SerGun78** [comment]
Sonny, hello!
Several releases have already been released.
The problem hasn't been resolved yet.
Is it too hard to fix?

**Comment by SerGun78** [comment]
Chacapama, this issue has already been confirmed by joomlaplates and sonvnn. There is still no solution.

**Comment by Chacapamac** [comment]
@SerGun78  can you point to me what you exactly want using my Astroid testing Website —> http://joom4-astro.graf-test.ca/blog
Sorry, I just have a bit of difficulty to understand what you exactly want to do....

**Comment by SerGun78** [comment]
[Chacapamac](https://github.com/Chacapamac), here https://github.com/templaza/astroid-framework/issues/48#issuecomment-991855663 and here https://github.com/templaza/astroid-framework/issues/48#issuecomment-993375265 he indicated what is necessary and how to correctly display the title in the blog and in a separate article

**Comment by felixkat** [comment]
Closing as referenced issue suggested it's not Astroid related

https://github.com/templaza/astroid-framework/issues/942
